src/main/scala/ai/privado/passes/PropertyParserPass.scala

```diff
@@ -67,7 +67,8 @@ class PropertyParserPass(
             FileExtensions.YAML,
             FileExtensions.YML,
             FileExtensions.XML,
-            FileExtensions.CONF
+            FileExtensions.CONF,
+            FileExtensions.JSON
```

Comment:
We have added Json for python property pass, This can have performance issues

Reply:
Yes removed.